.github/workflows/delete_review_app.yml

```diff
@@ -3,53 +3,51 @@ name: Delete review app on AKS
 on:
   pull_request:
     branches:
-    - main
+      - main
     types:
-    - closed
-
-permissions:
-  id-token: write
-  pull-requests: write
+      - closed
+  workflow_dispatch:
+    inputs:
+      pr_number:
+        description: PR number of review app to delete
+        required: false
+        type: string
 
 jobs:
   delete-review-app:
-    name: Delete Review App ${{ github.event.pull_request.number }}
-    concurrency: deploy_review_${{ github.event.pull_request.number }}
-    if: ${{ !contains(github.event.pull_request.labels.*.name, 'Dependencies') }}
+    name: Delete Review App ${{ github.event.pull_request.number || github.event.inputs.pr_number }}
+    concurrency: deploy_review_${{ github.event.pull_request.number || github.event.inputs.pr_number }}
     runs-on: ubuntu-latest
-    environment: review
-    steps:
-    - name: Checkout
-      uses: actions/checkout@v4
-      with:
-        token: ${{ secrets.GITHUB_TOKEN }}
-
-    - uses: hashicorp/setup-terraform@v3
-      with:
-        terraform_version: 1.9.8
-        terraform_wrapper: false
+    if: >
+      github.event.action == 'closed' ||
```

Comment:
L22 should probably be similar to previous delete workflow i.e.
${{ !contains(github.event.pull_request.labels.*.name, 'Dependencies') }}
as they don't use the deploy label in this repo
and 
(github.event.action == 'unlabeled' && github.event.label.name == 'deploy')
can be removed